projects/angular2-query-builder/src/lib/query-builder/query-builder.component.ts

```diff
@@ -167,7 +198,21 @@ export class QueryBuilderComponent implements OnInit, OnChanges, ControlValueAcc
 
   // ----------OnInit Implementation----------
 
-  ngOnInit() { }
+  ngOnInit() {
+    // Loop through all the fields and change the operators to value/label if they are just string arrays
+    for (const field in this.config.fields) {
+      if (this.config.fields[field].operators) {
+        this.config.fields[field].operators = (this.config.fields[field].operators as any[])
+        .map((operator: string | { value: string, label: string}) => {
+          if (typeof(operator) === 'string') {
+            return { value: operator, label: operator };
```

Comment:
I feel like the string array pattern should be deprecated. Should a console.warn be added here to note that?